src/core/pipe.go

```diff
@@ -11,57 +11,80 @@ import (
 	"context"
 	"sync"
 
+	log "github.com/sirupsen/logrus"
 	messagebus "github.com/vardius/message-bus"
 )
 
 const (
-	MessageQueueSize = 100
-	MaxPlugins       = 100
+	MessageQueueSize    = 100
```

Comment:
let's add a default value for this in defaults.go

Reply:
At the moment we don't configure these via the agent configuration file. So for now it might be best to leave as is and open a separate PR to handle how we configure the plugin pipeline. Just because this PR has a lot of changes already in it.